src/common/components/Recaptcha/RecaptchaWrapper.tsx

```diff
@@ -36,14 +37,21 @@ export const RecaptchaWrapper = ({
   buttonLabel,
   envService,
   $translate,
-  $log
+  $log,
+  $rootScope
 }: RecaptchaWrapperProps): JSX.Element => {
   const recaptchaKey = envService.read('recaptchaKey')
 
+  // Because the onSuccess callback is called by a React component, AngularJS doesn't know that an event happened and doesn't know it needs to rerender. We have to use $apply to ensure that AngularJS rerenders after the event handlers return.
```

Comment:
Can we break this comment into multiple lines?

```suggestion
  // Because the onSuccess callback is called by a React component, AngularJS doesn't know that an
  // event happened and doesn't know it needs to rerender. We have to use $apply to ensure that
  // AngularJS rerenders after the event handlers return.
```